doc/source/GettingStarted/ConfiguringGafferForThirdPartyTools/index.md

```diff
@@ -0,0 +1,194 @@
+# Configuring Gaffer for Third-Party Tools #
+
+Gaffer is compatible with the following commercial and open-source third-party tools:
+
+- [Appleseed](http://appleseedhq.net/)
+- [Arnold](https://www.solidangle.com/arnold/)
+- [3Delight](http://www.3delight.com/)
+- [Tractor](https://renderman.pixar.com/tractor)
+
+```eval_rst
+Gaffer comes with Appleseed, so it will require no additional configuration. For the rest of the tools in this list, you will need to set some additional :ref:`environment variables<Environment Variables>`.
+```
+
+> Note :
+> For the the following Linux instructions, we will assume you are using the _bash_ shell and are familiar with terminal commands. Other shells will have comparable methods for setting environment variables.
+
+
+## Configuring Gaffer for Arnold ##
+
+For Gaffer to load the GafferArnold module, an `ARNOLD_ROOT` environment variable must point to the Arnold installation directory. Before you begin, make sure that Arnold is correctly installed and configured, and close any open instances of Gaffer.
+
+
+### Arnold in Linux ###
+
+> Note :
+> For this instruction, we will assume you have Arnold !ARNOLD_VERSION! installed to `!ARNOLD_PATH_LINUX!`.
+
+To create the `ARNOLD_ROOT` environment variable in Linux:
+
+1. Open `~/.bash_profile` with a text editor.
+
+2. Add the line `export ARNOLD_ROOT=!ARNOLD_PATH_LINUX!` and save.
+
+3. In a terminal, reload the user configuration file, and then verify the variable is set:
+    
+    ```bash
+    user@desktop ~ $ source ~/.bash_profile
+    user@desktop ~ $ echo $ARNOLD_ROOT
+    !ARNOLD_PATH_LINUX!
+    ```
+
+4. The next time you start Gaffer, the Arnold nodes will be available from the node creation menu.
+    
+    <!-- TODO: ![Arnold node menu](images/arnoldNodes.png) -->
+
+
+### Arnold in OSX ###
```

Comment:
The section is indeed the same. This is part of the awkwardness of splitting style and content. Once the tabs are in, it will make more sense. I could comment it out for now, I suppose.

The Finder part is duplicated because documentation is never read serially. We can't assume whoever lands here will have read the articles leading up to it. There will be plenty more redundancy as this progresses...